nights/2024-04-12-README.md

```diff
@@ -1,26 +1,66 @@
 # 2024-04-12
 
-## Reviewed by:   your_name_here
+## Reviewed by:   Jake Mailhot
 
 ## Raw image gallery: http://physics.mnstate.edu/feder_gallery/2024-04-12
 
 ## Unusual images?
 
 Are there any images that look unusual? List the file name of any unusual images for this night here, with description:
 
-+ `crazy-object-001R.fit` -- not sure what this is an image of
-+ `another-crazy-object-003B.fit` -- this looks more like a flat than a science image.
-+ `full-moon-005B.fit` -- nice satellite or plane track in here.
+TIC_81247877-S001-R001-C001-rp.fit - slight drag on the image
+TIC_81247877-S001-R001-C007-rp.fit - Unclear
```

Comment:
for any marked "unclear" I would probably switch the reasoning to clouds- make the reasoning more direct